### PR TITLE
Clean up NetworkTables usage

### PR DESCRIPTION
**Pull request by auscompgeek**

### CODE REVIEW

**Comment by auscompgeek** [comment]
@ArthurAllshire seeing as you merged this into your branch, did you want to actually review this? :P

File: components/lifter.py [hunk 1]
```diff
@@ -100,14 +101,16 @@ def move(self, input_setpos):
         self.set_pos = input_setpos
         self.motor.set(self.motor.ControlMode.MotionMagic, self.metres_to_counts(self.set_pos))
 
-    def get_pos(self):
-        """Returns encoder position on lift
-
-       Returns:
-            int: The location of the lift
-        """
+    @magicbot.feedback
```
Comment (ArthurAllshire) [inline components/lifter.py:104]: Will turning this into a magic bot feedback not mean it is called every iteration, thus reading the sensor position every iteration, slowing down the code as it reads over the CAN bus?

File: components/lifter.py [hunk 2]
```diff
@@ -100,14 +101,16 @@ def move(self, input_setpos):
...
+    def get_pos(self) -> float:
+        """Get the current height of the lift."""
         return self.motor.getSelectedSensorPosition(0) / self.COUNTS_PER_METRE
 
+    @magicbot.feedback
```
Comment (ArthurAllshire) [inline components/lifter.py:109]: Ditto my above comment.

**Review by ArthurAllshire — CHANGES_REQUESTED** [review]
IDK how we do this - I don't think we can merge this PR without breaking mine. I think to make changes you should just push to my auto branch...

**Comment by auscompgeek** [comment]
This is already merged in your PR.

File: components/lifter.py (hunk 1 above)
Comment (auscompgeek) [reply components/lifter.py:104]: Yes. Yes it does. This is no worse than what it was doing before this change though.

**Pull request merged**